packages/dialtone-css/lib/build/less/components/empty-state.less

```diff
@@ -36,8 +36,10 @@
     padding: var(--dt-space-500) 0;
   }
 
+  /* stylelint-disable-next-line block-no-empty */
   &__content--md {}
 
+  /* stylelint-disable-next-line block-no-empty */
   &__content--lg {}
```

Comment:
I'd prefer to remove empty blocks rather than disabling stylelint if possible.

Reply:
removed